apps/rig_inbound_gateway/lib/rig_inbound_gateway_web/connection_init.ex

```diff
@@ -64,4 +62,27 @@ defmodule RigInboundGatewayWeb.ConnectionInit do
         on_error.("Subscription denied (not authorized).")
     end
   end
+
+  # ---
+
+  @spec subscriptions_query_param_to_body(map) :: {:ok, String.t()} | {:error, String.t()}
+  def subscriptions_query_param_to_body(query_params)
```

Comment:
Should be perhaps `subscriptions_query_params_to_body`? Since it's accepting multiple params not just single one.